includes/shortcodes.php

```diff
@@ -188,7 +188,7 @@ function give_form_shortcode( $atts ) {
 	ob_start();
 
 	if ( ! FormUtils::isLegacyForm( $formId ) ) {
-		$showIframeInModal = 'button' === $atts['display_style'];
+		$showIframeInModal = 'button' === $atts['display_style'] || 'modal' === $atts['display_style'];
```

Comment:
Let's add an `unreleased` tag in the top of this function to document this change.

Reply:
Resolved: 5f92dea8b1d318cae9fb044753e3f6b6517f86a2